src/platform/monitoring/sentry.js

```diff
@@ -34,7 +34,7 @@ if (trackErrors) {
       if (evt && evt.reason) {
         Sentry.captureException(evt.reason);
       } else {
-        Sentry.captureMessage('Unhandled promise rejection');
+        Sentry.captureException('Unhandled promise rejection');
```

Comment:
would like a follow-up on if this worked or not and what the difference between `captureMessage` and `captureException` are. looked at the docs and didn't see anything that implies this is working differently.

this `unhandledrejection` event- does it often happen with an `event.reason`? It doesn't have great browser support: https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event - wonder if we could enforce this with linting e.g. if you don't have a fallback defined for a promise tree then it throws an error. 

do we similarly capture `onerror` (maybe sentry does this automatically).

Reply:
So you make a good point. I had this whole long answer written out before I read [this](https://docs.sentry.io/platforms/javascript/#automatically-capturing-errors-with-promises).

So....I think instead, I'ma nuke this whole event handler (after testing that it behaves as expected, of course).

Reply:
![image](https://user-images.githubusercontent.com/12970166/60058207-60867580-969c-11e9-96cb-964f45d674e8.png)

Turns out it's unnecessary after all. I removed the `localhost` condition so it _did_ try to send events to Sentry on localhost, removed that whole event handler dealio, and did one of these numbers immediately after the `Sentry.init()`:
```javascript
(function reject() {
  return Promise.reject();
})();
```

Reply:
@cvalarida Might also be good to test this in IE11, where Promise is polyfilled, to make sure Sentry picks those unhandled rejections up.

Reply:
Ooh, good call! :+1:

Reply:
Confirmed! I hosted `vets-website` on my local network and accessed it from IE 11. It attempted to make the call to Sentry.